### PR TITLE
slider added

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot:
ezgif com-gif-maker

## Deploy Preview Link:
https://deploy-preview-15--comp-ui.netlify.app/components/slider/slider

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
Looks good to go !

File: components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.slider-wrap {
+    position: relative;
+    margin: 0 3rem;
+}
+.slider {
+    width: 100%;
+}
+.bubble {
+    background: var(--primary-color);
+    color: white;
```
Comment (xii-vi) [inline components/slider/slider.css]: Use variable here.

**Review by xii-vi — APPROVED** [review]
_added a little comment. Check it out.
Rest is okay to merge._

File: components/slider/slider.css (hunk 1 above)
Comment (bishnoimukesh) [reply components/slider/slider.css]: done

**Pull request merged**